src/eos-updater.c

```diff
@@ -144,6 +163,8 @@ on_bus_acquired (GDBusConnection *connection,
       g_signal_connect (updater, "handle-apply", G_CALLBACK (handle_apply), local_data->data);
     }
 
+  g_signal_connect (updater, "handle-cancel",  G_CALLBACK (handle_cancel), local_data->data);
```

Comment:
I think this should be conditional like the other method callbacks, so that on a live boot it will return `EOS_UPDATER_ERROR_LIVE_BOOT` rather than succeeding (and doing nothing).

Reply:
I did it like this because nothing special will happen (it'll just cancel the GCancellable...). But I agree that for consistency it's better to have this function as the other methods. Will change.

Reply:
Yeah, I can see why you did it; but I think consistency is more important here. Thanks.